### PR TITLE
Update govuk_elements to version 2.2.1

### PR DESCRIPTION
**Pull request by Guntrisoft**
**Forms with radio buttons before**
screen shot 2016-12-02 at 16 58 55

**Forms with radio buttons after**
screen shot 2016-12-02 at 17 00 07

Blog post explaining the change of style for radio/checkboxes
https://designnotes.blog.gov.uk/2016/11/30/weve-updated-the-radios-and-checkboxes-on-gov-uk/

- made some minor adjustments to form to look good with
  new radio/checkbox styles
- removed field_with_errors wrapping div which
  conflicts with new radio/checkbox JS modules

Full list of changes:

Remove the images path override from the helpers partial
https://github.com/alphagov/govuk_elements/pull/292

Bump govuk frontend toolkit to 4.16.1
https://github.com/alphagov/govuk_elements/pull/288

Form validation patterns for conditionally revealing content
https://github.com/alphagov/govuk_elements/pull/286

Centre text on full-width buttons
https://github.com/alphagov/govuk_elements/pull/289

Lint JS code using StandardJS
https://github.com/alphagov/govuk_elements/pull/290

Update govuk_frontend_toolkit to 5.0.0 and govuk_template to 0.19.0
https://github.com/alphagov/govuk_elements/pull/333

Add a select box example
https://github.com/alphagov/govuk_elements/pull/303

Add bullet points describing use of disabled buttons
https://github.com/alphagov/govuk_elements/pull/304

Add reasoning for native over custom file inputs

Clear floats on details elements
https://github.com/alphagov/govuk_elements/pull/328

Add a .bold utility class
https://github.com/alphagov/govuk_elements/pull/321

Remove external link styles
https://github.com/alphagov/govuk_elements/pull/274

Remove rounded corners on form inputs elements and textareas in iOS
https://github.com/alphagov/govuk_elements/pull/342

Fix focus outline on form fields in Chrome / Safari
https://github.com/alphagov/govuk_elements/pull/346

Updates the contribution guidelines
https://github.com/alphagov/govuk_elements/pull/339

Recommend .visually-hidden over .visuallyhidden
https://github.com/alphagov/govuk_elements/pull/341

Add snippets for data visualisation examples
https://github.com/alphagov/govuk_elements/pull/350
https://github.com/alphagov/govuk_elements/pull/351

Fix the grey left hand border example for Radios and checkboxes
https://github.com/alphagov/govuk_elements/pull/349

Fix the spacing underneath the "inline" block label example
https://github.com/alphagov/govuk_elements/pull/348

Custom radios / checkboxes
https://github.com/alphagov/govuk_elements/pull/296

Fix contrast issues with phase banners. Use $govuk-blue for the phase tag and update the examples
https://github.com/alphagov/govuk_elements/pull/364

### CODE REVIEW

**Comment by Guntrisoft** [comment]
@benbarnett @AndrewVos Give this a quick review once again, now that various test issues have been fixed

File: app/assets/javascripts/checkbox-radio-init.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/* global GOVUK */
+$(function() {
+  'use strict';
+
+  jQuery.fx.off = true;
```
Comment (benbarnett) [inline app/assets/javascripts/checkbox-radio-init.js]: Is this necessary here?

File: config/application.rb [hunk 2]
```diff
@@ -23,5 +23,9 @@ class Application < Rails::Application
     config.middleware.use BounceBrowserconfig
 
     config.mount_javascript_test_routes = false
+
+    config.action_view.field_error_proc = proc { |html_tag, _|
```
Comment (benbarnett) [inline config/application.rb]: I think this might be worth breaking into it's own commit to help explain what it does and why the change was made?

Comment (Guntrisoft) [inline config/application.rb]: will do

File: features/pages/booking_step_two.rb [hunk 3]
```diff
@@ -20,5 +20,17 @@ class BookingStepTwo < SitePrism::Page
 
     element :submit, '.t-submit'
     element :back, '.t-back'
+
+    def check_hidden_checkbox(element, check)
```
Comment (Guntrisoft) [inline features/pages/booking_step_two.rb]: done

File: config/application.rb (hunk 2 above)
Comment (Guntrisoft) [inline config/application.rb]: done

Comment (benbarnett) [inline config/application.rb]: Can we add a bit more of a 'why this was changed' thing to that commit msg?

Comment (Guntrisoft) [inline config/application.rb]: sorry! - done!
